### ISSUE TITLE
Add documentation for scopes?

### ISSUE THREAD

**Issue by paperscissors** [adam-paterson/oauth2-stripe#5]
Took a bit to figure out how to change these settings when running the OAuth request; actually even PHP League's base package doesn't mention this. I'll reach out them as well, but I had to look at other libraries to find the method: https://github.com/thephpleague/oauth2-linkedin#managing-scopes

```
$options = [
    'scope' => ['abc','xyz'] // array or string
];

$authorizationUrl = $provider->getAuthorizationUrl($options);
```

Can be pretty important depending on what scopes you need to use with the OAuth + Stripe.

**Comment by adam-paterson** [comment]
@paperscissors Good point, I'll add the scope docs at some point today. Unless you have time to submit a PR.
